forgerock-openbanking-uk-aspsp-as/forgerock-openbanking-uk-aspsp-as-gateway/forgerock-openbanking-uk-aspsp-as-gateway-server/src/test/java/com/forgerock/openbanking/aspsp/as/api/authorisation/AuthorisationApiControllerTest.java

```diff
@@ -62,96 +70,126 @@
     private AMGatewayService amGatewayService;
     @Mock
     private DiscoveryConfig discoveryConfig;
-    @InjectMocks
-    private AuthorisationApiController authorisationApiController;
+    @Mock
+    private JwtOverridingService jwtOverridingService;
+    @Mock
+    private TokenUsageService tokenUsageService;
+
+    private AuthorisationApiController authorisationApiController ;
+
+    @Before
+    public void setUp() throws Exception{
```

Comment:
Minor - this doesn't throw an exception.

Reply:
Fixed